server/app/views/social_links/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>SocialLinks#edit</h1>
```

Comment:
We are using a react frontend - these erb files are unnecessary

Reply:
They were automatically generated...